packages/client/components/PokerEstimateHeaderCardJira.tsx

```diff
@@ -52,7 +52,7 @@ const CardDescription = styled('div')<{isExpanded: boolean, maxHeight: number}>(
   fontSize: 14,
   margin: 0,
   maxHeight: isExpanded ? maxHeight : 30,
-  overflow: 'hidden',
+  overflowY: 'scroll',
```

Comment:
Use `auto` instead of `scroll` because for some operating systems `scroll` will force the scrollbar to appear even when it doesn’t need to, and it’s clunky. To my knowledge, `auto` only adds the scrollbar when it’s needed.